src/components/Book.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable react/prop-types */
```

Comment:
- Please remove all disabled linters from your codebase, linters help us ensure we write code that is clean and easy to maintain. You can check [here](https://reactjs.org/docs/typechecking-with-proptypes.html) for more information on how to fix this error .Please fix this 🙏